templates/ts-react-apollo-client/.graphqlrc.yml

```
@@ -1,7 +1,7 @@
 ## GraphQL Config Generated by Graphback
 ## Please review configuration and adjust it for your own project
-schema: ./schema.graphql
-documents: ./src/graphql/**/*.graphql
+schema: ../src/schema/schema.graphql
+documents: ./src/graphql/**/**.graphql
```

```suggestion
documents: ./src/graphql/**/*.graphql
```

Should this be kept as it was? Not sure if that would make a difference or work.

---

good catch.